services/attachment/attachment.go

```diff
@@ -48,6 +61,10 @@ func UploadAttachment(ctx context.Context, file io.Reader, allowedTypes string,
 		return nil, err
 	}
 
+	if maxFileSize >= 0 && fileSize > maxFileSize {
+		return nil, ErrAttachmentSizeExceed{MaxSize: maxFileSize, Size: fileSize}
```

Comment:
fileSize looks like an user input - how do we make sure that it's not borked?

Reply:
I think we should wrap `file io.Reader` into an limited reader, as copyN does:
https://cs.opensource.google/go/go/+/refs/tags/go1.25.1:src/io/io.go;l=363-373
```
file = io.LimitReader(file, maxFileSize+1024) // add 1k more to allow sidecases where encoding tells us its exactly maxFileSize but the actuall created file is >maxFileSize ...